src/ServicePulse.Host/app/js/services/service.formatter.js

```diff
@@ -4,22 +4,32 @@
     function formatter() {
         var secondDuration = moment.duration(1000);
         var minuteDuration = moment.duration(60 * 1000);
-        var hourDuration = moment.duration(60 * 60 * 1000);
+        var hourDuration = moment.duration(60 * 1000); //this ensure that we never use minute formatting
```

Comment:
How does making the hour duration the same length as a minute ensure we never use minute formatting? Shouldn't it be the other way around (i.e. make minute duration the same as hour duration)?

Reply:
Our chose way of formatting was to use hour formatting starting from a value of 1 minute. So for 1 minute we display: 00:01. 
The fact that those 2 values are equal (minute and hour duration) causes that minute duration will never be used.